test/fixtures/samples/es7-object-spread/contrived.bad.js

```diff
@@ -0,0 +1,7 @@
+// Made up sample.. not genuine
```

Comment:
Could you add to the `contrived.good.js` examples of usages where Object.assign is still allowed? e.g.:

```js
const a = { b: 1 };

Object.assign(a, { c: 2 });
```